tests/integration_tests/test_experiment.py

```diff
@@ -41,9 +41,8 @@
 
 from tests.integration_tests.utils import date_feature
 
+
 # The following imports are pytest fixtures, required for running the tests
-from tests.fixtures.filenames import csv_filename
-from tests.fixtures.filenames import yaml_filename
```

Comment:
please readd both lines, pycharm code optimization removed it because it's not referenced, but is actually needed

Reply:
Added the lines again.